README.md

```diff
@@ -1,44 +1,39 @@
 # Katib
 
 Hyperparameter Tuning on Kubernetes.
-This project is [Google vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf) inspired.
-Katib is a scalable and flexible hyperparameter tuning framework and  tightly integrate with kubernetes.
-And it does not depend on a specific DL framework.
-There are examples of three frameworks ( tensorflow, mxnet, and  pytorch).
+This project is inspired by [Google vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf).
+Katib is a scalable and flexible hyperparameter tuning framework and is tightly integrated with kubernetes.
+Also it does not depend on a specific Deep Learning framework e.g. TensorFlow, mxnet, and  pytorch).
```

Comment:
s/mxnet/MXNet
s/pytorch/PyTorch

Reply:
good point ... I was just cleaning basic typos ... will do it

Reply:
Thanks :smile: